src/main/java/com/adscore/signature/sample/SignatureVerifierSampleMain.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2020 AdScore Technologies DMCC [AE]
+ *
+ * Licensed under MIT License;
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.adscore.signature.sample;
+
+import com.adscore.signature.SignatureVerificationResult;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SignatureVerifierSampleMain {
+
+  private static final Map<String, Sample> samples;
+
+  static {
+    samples = new HashMap<>();
+
+    ConstantArgumentsBaseSample baseSample = new ConstantArgumentsBaseSample();
+    samples.put(baseSample.getName(), baseSample);
+  }
+
+  public static void main(String[] args) {
+
+    if (args == null || args.length == 0 || samples.get(args[0]) == null) {
+      throw new SignatureVerifierSampleException(
+          "Not found sample for given sample name. Check README.md or provide correct sample name as a first argument.");
```

Comment:
what does that mean ?:)

Reply:
To execute any sample, the first java parameter should be the name of the sample.

```
java -jar build/libs/client-libs-java-samples-1.0.4.jar all-args-sample {arg2} ... {argn...} 
```

If user put:
```
java -jar build/libs/client-libs-java-samples-1.0.4.jar all-args-sample some-not-existing-sample-name {arg2} ... 
```
Then this exception is thrown.  (`samples.get(args[0]) == null` -> Sample does not exist in a sample map)